packages/components/src/components/EmptyContent/EmptyContent.tsx

```diff
@@ -0,0 +1,43 @@
+import { ddsBaseTokens } from '@norges-domstoler/dds-design-tokens';
+import { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { Typography } from '../Typography';
+
+const { colors } = ddsBaseTokens;
+
+const StyledEmptyContent = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 10rem;
+  height: 100%;
+  width: 100%;
+  background-color: ${colors.DdsColorNeutralsGray1};
+  padding: 1.5rem;
```

Comment:
Bruk gjerne design-tokens her
```suggestion
  padding: ${spacing.SizesDdsSpacingLayoutX15};
```